apps/src/templates/rubrics/EvidenceLevelsForTeachersV2.jsx

```diff
@@ -29,7 +29,7 @@ export default function EvidenceLevelsForTeachersV2({
   // around evidence level.
   const passFail = useMemo(() => {
     if (!!aiEvalInfo) {
-      if (understanding === INVALID_UNDERSTANDING) {
+      if (understanding !== INVALID_UNDERSTANDING) {
```

Comment:
I am sure this is intentional... but can you explain the reasoning for the change?

Reply:
Had it the wrong way before, so it was not showing the correct style when no understanding was selected.